lib/repl.js

```diff
@@ -361,7 +361,7 @@ function REPLServer(prompt,
 
     // Check to see if a REPL keyword was used. If it returns true,
     // display next prompt and return.
-    if (cmd && cmd.charAt(0) === '.' && isNaN(parseFloat(cmd))) {
+    if (cmd && isNaN(parseFloat(cmd)) && /^\.[a-z]+\s*[^(]*$/.test(cmd)) {
       var matches = cmd.match(/^\.([^\s]+)\s*(.*)$/);
```

Comment:
`/^\.[a-z-_]+\s*[^(){}[];=|&]*$/` doesn't allow file names with the special characters`^(){}[];=|&`. I can live that.